anaSBitReadout.py

```diff
@@ -312,16 +322,16 @@
             # filling ieta 1Dhistos
             ieta_h_strip[eta].Fill((phi-1)*128+strip[0])
```

Comment:
Likewise here, `128` is `CHANNELS_PER_VFAT`

Reply:
done